### ISSUE TITLE
Does not work with form-inline from bootstrap

### ISSUE THREAD

**Issue by pacificobr** [snapappointments/bootstrap-select#1969]
When I use the class form-inline in a form the select does not follow this configuration.

```
<form class="form-inline">
   <select class="selectpicker">
     <option>Mustard</option>
     <option>Ketchup</option>
     <option>Barbecue</option>
   </select>
</form>
```
And it gets worse if I use _multiple_ (select with multiple values)

I'd really love to use this plugin, but this problem doesn't allow me.

Best regards.
[Brazil]

**Comment by caseyjhol** [comment]
In the meantime, setting `data-width="auto"` or `data-width="fit"` on the select should fix your problem.

**Comment by caseyjhol** [comment]
Released in [v1.13.8](https://github.com/snapappointments/bootstrap-select/releases/tag/v1.13.8)!
